src/pages/api/venues/[slug].js

```diff
@@ -0,0 +1,29 @@
+export default async function handler(request, response) {
+  try {
+    const baseURL = "https://app.ticketmaster.com/discovery/v2/";
+    const { slug, ...query } = request.query;
+    const url =
+      baseURL +
+      slug +
+      "?" +
+      new URLSearchParams({
+        apikey: process.env.API_KEY,
+        ...query,
+      }).toString();
+
+    const fetchData = await fetch(url);
```

Comment:
I would expect `fetchData` to be a function instead of a variable; what about `fetchedData`?

Reply:
✅️